.github/workflows/kindIntegTest.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         - webhook_validation
         # Three worker tests:
         - canary_upgrade
-        - config_change_condition
+        # - config_change_condition # config_change takes care of testing the same
```

Comment:
Maybe you could remove the line altogether and the corresponding test file?
Is there a reason to keep them?

Reply:
I think that should be done in a separate PR, since at the moment there's lots of those tests we've merged together. I think all of those should be cleaned up in a single PR and that way we would have a single commit which could then be used to track "old tests" if ever necessary.